quarkus-integration/quarkus-benchmark/integration-test/pom.xml

```diff
@@ -81,4 +81,42 @@
       </plugin>
     </plugins>
   </build>
+
+  <profiles>
+    <profile>
+      <!--
+        Benchmarker is not supported in native.
+        However, Quarkus 3.2.4+ seems to have an issue where the native profile,
+        if present on another unrelated module, is magically and wrongly inherited in this module.
+        This means the native compilation can not be avoided and needs to be made to pass.
+        Tests at runtime are then disabled.
+      -->
+      <id>native</id>
+      <activation>
+        <property>
+          <name>native</name>
+        </property>
+      </activation>
+      <dependencies>
+        <dependency>
+          <groupId>org.apache.logging.log4j</groupId>
+          <artifactId>log4j-1.2-api</artifactId>
+        </dependency>
+      </dependencies>
+      <dependencyManagement>
+        <dependencies>
+          <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-1.2-api</artifactId>
+            <version>${version.org.apache.logging.log4j}</version>
+          </dependency>
+        </dependencies>
+      </dependencyManagement>
+      <properties>
+        <quarkus.package.type>native</quarkus.package.type>
+        <quarkus.native.additional-build-args>--initialize-at-run-time=freemarker.ext.jython.JythonWrapper</quarkus.native.additional-build-args>
```

Comment:
Good as a temporary workaround, but please at least file an issue to investigate this further. Ideally, this should be reported to Quarkus.

If this module is not supposed to work in native, we should not lose time by native compilation in every CI run.

Reply:
Agreed off-line that no issue will be filed.